plugins/status.yaml

```diff
@@ -43,10 +43,13 @@ spec:
     bin: "status.exe"
   shortDescription: Show status details of a given resource.
   homepage: https://github.com/bergerx/kubectl-status
-  description: |
-    Show status details of a given resource. Most useful when debugging Pod issues.
+  caveats: |
+    Usage:
+      $ kubectl status
```

Comment:
@bergerx I recommend removing this. We already show it upon installation so it's actually a duplicate right now.